kinova_driver/src/kinova_arm.cpp

```diff
@@ -438,7 +438,7 @@ void KinovaArm::ovisJointVelocityCallback(const ovis_msgs::OvisJointVelocity::Co
       trajectory_point_velocity.Position.Actuators.Actuator1 = INVERSE * msg->joint_velocity * number_of_degree_per_sec;
       break;
     case 1:
-      trajectory_point_velocity.Position.Actuators.Actuator2 = INVERSE * msg->joint_velocity * number_of_degree_per_sec;
+      trajectory_point_velocity.Position.Actuators.Actuator2 = msg->joint_velocity * number_of_degree_per_sec;
```

Comment:
Here, only the join 1 and 4 are reversed. In `void KinovaArm::publishJointAngles(void)`, the join 1, 3 and 4 are inverted.
Is there a reason? it seems odd. Can someone confirm this?

Reply:
It's odd indeed, good catch, this is probably hidden by the UI sending a reverse command, we might want to fix all those inversions directly in the urdf in the future, however it might be too time consuming for the available ressources before the competition

Reply:
Interesting, I don't really understand how it works then lol. As for the UI it always sends positive joint angles `[180,180,180,180,180,180]` for the right angle position.